src/categories.js

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+import { getCategories } from "./api";
+import { useEffect, useState } from "react";
+
+const Categories = () => {
+  const [categories, setCategories] = useState([]);
+
+  useEffect(() => {
+    getCategories().then((res) => {
+      setCategories(res.categories);
+    });
+  }, []);
+
+  return (
+    <ul className="ulcategories">
+        <li><a class="active" href="#home"><Link to="/" >Home</Link></a></li>
```

Comment:
Couple of errors here from nesting <a> tag (DOM attribute) and Links, highly preferable to just use the link instead, you can still apply class stylings using the className="" syntax!